docs/decisions/0002-auto-enrollment-post-request-approval.rst

```diff
@@ -0,0 +1,113 @@
+2. Automatically Enrolling Learner After a Subsidy Request Approval
+===================================================================
+
+
+Status
+======
+
+?
+
+Context
+=======
+
+Learners are currently able to request a subsidy (license or coupon code) through the enterprise-access service.
+After a request is approved, the learner still has to redeem the subsidy and enroll in a course manually.
+Since the goal of a learner is to access a course, we want to automatically redeem the subsidy and enroll
+them after a request approval.
+
+For coupon code requests, requiring manual redemption is not ideal. We cannot that the coupon code is only used
+for the course it was requested for, and have no good way of tracking the usage of the coupon code.
+
```

Comment:
Should we explicitly add a line here about license subsidy_request approval for thoroughness? (Even if this event-bus stuff doesn't directly impact the flow)